base/cvd/fastboot/fs.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
nit: the original file doesn't have a license, should this file have one?

Reply:
Done